### PR TITLE
! fix load! merging data

### PR DESCRIPTION
**Pull request by pape-io**
Fix https://github.com/local-ch/lhs/issues/43

### CODE REVIEW

**Comment by kot-begemot** [comment]
:+1: looks good

File: spec/proxy/load_spec.rb [hunk 1]
```diff
@@ -32,14 +32,14 @@ class SomeService < LHS::Service
   context 'load' do
 
     it 'is loading data remotely when not present yet' do
-      link.load!.id
+      expect(link.load!.id).to be
+      expect(link.id).to be
```
Comment (acoiro) [inline spec/proxy/load_spec.rb]: this means present?

**Comment by acoiro** [comment]
:+1:

File: spec/proxy/load_spec.rb (hunk 1 above)
Comment (pape-io) [inline spec/proxy/load_spec.rb]:
```
 expect(obj).to be         # passes if obj is truthy (not nil or false)
```
https://www.relishapp.com/rspec/rspec-expectations/v/3-3/docs/built-in-matchers/be-matchers

**Pull request merged**